files/similar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "similar",
+  "repo": "InputSimilar/similar",
+  "files": {
+    "include": ["similar.eva.js"]
```

Comment:
This file doesnt exist here https://github.com/InputSimilar/similar/tree/similar